packages/order-utils/src/asset_data_utils.ts

```diff
@@ -1,5 +1,8 @@
 import {
     AssetProxyId,
+    ERC1155AssetData,
```

Comment:
You'll need to reexport these in the `index.ts` of this package and 0x.js (see doc-gen failing tests).